extensions-core/histogram/src/test/java/org/apache/druid/query/aggregation/histogram/sql/FixedBucketsHistogramQuantileSqlAggregatorTest.java

```diff
@@ -206,7 +206,12 @@ public void testQuantileOnFloatAndLongs() throws Exception
                        + "FROM foo";
 
     // Verify results
-    final List<Object[]> results = sqlLifecycle.runSimple(sql, QUERY_CONTEXT_DEFAULT, authenticationResult).toList();
+    final List<Object[]> results = sqlLifecycle.runSimple(
+        sql,
+        QUERY_CONTEXT_DEFAULT,
+        ImmutableList.of(),
```

Comment:
Adding a named constant (e.g., `NO_PARAMETERS`) may be useful here.

Reply:
done